frontend/js/behaviors/core/stickySidebar.js

```diff
@@ -204,6 +204,9 @@ const stickySidebar = function(container){
     window.addEventListener('scroll', handleScroll);
     window.addEventListener('resized', handleResize);
 
+    document.addEventListener('accordion:toggled', () => {
```

Comment:
For whatever reason this needs to be a functional call in order to update the state